contrib/node/examples/src/node/hello-test/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think we still want to allow node developers to develop. If we don't allow node developers to develop in place. If pants is incompatible with all the other tons of dev tools, it's very easy to see that between pants and other tools developers will choose other tools.